turbinia/workers/__init__.py

```diff
@@ -197,6 +201,39 @@ def set_error(self, error, traceback_):
     self.error['error'] = error
     self.error['traceback'] = traceback_
 
+  def serialize(self):
+    """Prepares result object for serialization.
+
+    Returns:
+      dict: Object dictionary that is JSON serializable.
+    """
+    self.run_time = self.run_time.total_seconds() if self.run_time else None
+    self.start_time = str(self.start_time)
+    self.input_evidence = [x.serialize() for x in self.input_evidence]
```

Comment:
See comment below, but input_evidence is now just one Evidence object and not a list.